framework/scenario/src/api/local_api_vh/static_var_api_vh.rs

```diff
@@ -82,20 +82,18 @@ impl<VHB: VMHooksApiBackend> StaticVarApiImpl for VMHooksApi<VHB> {
     }
 
     fn is_scaling_factor_cached(&self, decimals: usize) -> bool {
```

Comment:
I think this will be confusing for future developers. We should make this just a getter, and call the setter explicitly from ManagedDecimal.

Performance contraints don't seemt to justify it.